### ISSUE TITLE
support for extracting the name from Term.ApplyUnary

### ISSUE THREAD

**Issue by xeno-by** [scalameta/scalameta#280]

**Comment by xeno-by** [comment]
On a second thought, this is not a big deal for now, because an interpreter can call `.desugar`, which will transform stuff like `+expr` into `expr.unary_+`, and we now how to extract a name from there.

This is still a blocker for hygienic name manipulation, when we want to extract the name of a unary call and insert it elsewhere, but we're too far away from hygiene (#156) at the moment to seriously consider this being an important issue.

**Comment by xeno-by** [comment]
@olafurpg and I had a discussion, and we decided to pronounce quasiquotes feature-complete.

There are still some open issues (#46, #223, #251, #280 and others), but they all have two things in common: 1) they require non-trivial design work to be fixed, 2) they aren't popular at all. Given our recent decision to start avoiding unwarranted breaking changes in 2.0, we'll consider quasiquotes good enough at this point and will close all outstanding feature requests.

We will still be fixing bugs (in fact, I've fixed about a dozen of such bugs for 2.0), but without strong evidence our design efforts will be directed elsewhere.
